cloudmapper.py

```diff
@@ -95,6 +112,10 @@ def run_prepare(arguments):
                         required=False, type=str)
     parser.add_argument("--regions", help="Regions to restrict to (ex. us-east-1,us-west-2)",
                         default=None, type=str)
+    parser.add_argument("--vpc-ids", help="VPC ids to restrict to (ex. us-east-1,us-west-2)",
```

Comment:
Please change the examples here from `us-east-1,us-west-2` to vpc id and vpc names, such as `vpc-12345678` and `prod`.